src/main/java/uk/gov/hmcts/reform/hmc/service/HearingManagementServiceImpl.java

```diff
@@ -181,6 +190,27 @@ public GetHearingsResponse getHearings(String caseRef, String status) {
         return getHearingsResponseMapper.toHearingsResponse(caseRef, entities);
     }
 
+    /**
+     * get parties notified.
+     * @param hearingId hearing id
+     * @return  list dateTimes
+     */
+    @Override
+    public List<LocalDateTime> getPartiesNotified(Long hearingId) {
+        isValidFormat(hearingId.toString());
+        if (!hearingRepository.existsById(hearingId)) {
+            throw new HearingNotFoundException(hearingId);
+        }
```

Comment:
```suggestion
        validateHearingId(hearingId);
```
Can we re-use this existing method which covers all the standard checks?